Makefile

```diff
@@ -114,4 +114,11 @@ clean:
 	rm -rf $(UPTODATE_FILES) $(EXES) $(PROTO_GOS)
 	go clean ./...
 
+update-gazelle:
+	gazelle -go_prefix github.com/weaveworks/cortex -external vendored
 
+bazel:
+	bazel build //cmd/...
+
+bazel-test:
```

Comment:
> When would a Cortex developer want to run Bazel? How does this PR actually help?

You may want to use bazel locally to build & test, as for incremental builds & tests it faster.